### PR TITLE
[Feature] Added Bot Status 

### PR DESCRIPTION
**Pull request by vind3v17**
## Fixes Issue (#25 )

This PR fixes the following issues :

Added a Status/Activity for the Bot (LISTENING to cm!help)

## Check List (Check all the boxes which are applicable)

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagarised content.
- [ ] The title of my pull request is a short description of the requested changes.

## Screenshots 
Add all the screenshots which support your changes

### CODE REVIEW

**Comment by kaiwalyakoparkar** [comment]
hey @vind3v17, why did you close it? Do you need any help? You can do changes in open PR as well :slightly_smiling_face:

**Comment by vind3v17** [comment]
@kaiwalyakoparkar oh I see. I didnt knew about it. Thankyou for the info.
Actually I wanted to change the name because its repeating `to`

**Comment by kaiwalyakoparkar** [comment]
@vind3v17 Yeah! Don't worry. When it's complete ping me. One more thing as it is under development. You can find an option to convert it into a `Draft PR`

**Comment by vind3v17** [comment]
@kaiwalyakoparkar Yea Sure! Its already completed though.

File: Commands/presence.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module.exports = async function (client) {
+	client.user.setPresence({
```
Comment (kaiwalyakoparkar) [inline Commands/presence.js:2]: The indentation of the code is not correct. Please indent your code as shown in the image. Indent `client.user.setPresence`

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]
Please do the following indentation changes

File: Commands/presence.js (hunk 1 above)
Comment (vind3v17) [reply Commands/presence.js:2]: Yes Sure. On it!

Comment (vind3v17) [reply Commands/presence.js:2]: Yes Sure. On it!

**Comment by vind3v17** [comment]
@kaiwalyakoparkar Its fixed now! 😅

**Review by kaiwalyakoparkar — APPROVED** [review]
LGTM :rocket: Thanks for collaborating and contributing :smile:

**Pull request merged**